src/app/pages/system/general-settings/localization-form2/components/value-accessor-parent.class.ts

```diff
@@ -0,0 +1,27 @@
+import { ControlValueAccessor } from '@angular/forms';
+
+export class ValueAccessorParent implements ControlValueAccessor {
```

Comment:
I would prefer not to use inheritance and just have this code in each component.
value will not always be string, you'd have to reimplement this with generics, there may also be different functionality needed for example on `writeValue` and in general inheritance will make it harder to navigate.